requirements.txt

```diff
@@ -1,13 +1,14 @@
 asgiref==3.5.2
 Django==4.1.3
 djangorestframework==3.14.0
+djangorestframework-simplejwt==5.2.2
 pytz==2022.6
 sqlparse==0.4.3
 tzdata==2022.6
 django-phonenumber-field==7.0.0 # enables localization of phone number based on couuntry code
 phonenumbers==8.13.0
 Pillow==9.3.0
 drf-nested-routers
-djangorestframework_simplejwt
 model_bakery
-pytest-django
+pytest-django==4.5.2
+black==22.12.0 # used for formatting code
```

Comment:
could we setle on flake8 instead???

Reply:
I wanted black to sort out the formatting issues I saw after @OyugoObonyo push his PR